Development Merge: latest dev into stage for latest deployment of enhancement

supriyartambewaiin opened the pull request:
## Screenshots
: - No Screenshots

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Enhanced chat functionality with image attachments and improved comment handling.
  - Redesigned layout with modular top navigation and sidebar for a streamlined navigation experience.

- **Style**
  - Updated visual styling across components, including refined spacing, theming improvements, and refreshed iconography.
  - Reorganized component groupings in the design library for clearer categorization.

- **Chores**
  - Upgraded dependencies and tooling for improved performance and compatibility.
  - Modernized configuration and build scripts to align with the latest standards.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

agaurav0019 reviewed (approved):
Changes reviewed and approved.

sharmaamanwai committed:
Refactor side navigation styles, update Storybook configuration, and enhance component prop types; update package dependencies and scripts for improved compatibility.

nknikhilkasar committed:
Refactor CSS styles and enhance AI chat bot component functionality

nknikhilkasar committed:
Update AI chat bot and typing section to support image input in message sending

nknikhilkasar committed:
Remove border from chat input in sidebar CSS

nknikhilkasar committed:
Replace attachment icon with enhanced RdsAttachment component in typing section

nknikhilkasar committed:
Refactor story titles and clean up whitespace in AI chat bot, typing section, and subscription plan components

supriyartambewaiin committed:
Merge branch 'development' into users/nikhil/task-1691-Create-new-component-AI-Chat-Bot-as-per-design

supriyartambewaiin committed:
Merge branch 'development' into users/nikhil/task-1691-Create-new-component-AI-Chat-Bot-as-per-design

rameshwarkamandar committed:
Push my code css changes is pending share branch to dipak

rameshwarkamandar committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

Dipak7028 committed:
Resolved Bug :1763 Design System Component Cache Long Cache text are hidden behind delete button in mobile views

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1763-components-cache-long-cache-text-are-hidden-behind-delete-button-in-mobile-views

Dipak7028 committed:
Update rds-comp-application-workflows.tsx

sharmaamanwai committed:
Merge branch 'development' into feature/node_upgrade

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1760-components-application-work-flows-UI-issue-in-small-mobile-view

Dipak7028 committed:
Update rds-comp-grid.tsx

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/task-1691-Create-new-component-AI-Chat-Bot-as-per-design

nknikhilkasar committed:
Add attachment styling and adjust layout for typing section

Dipak7028 committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

nknikhilkasar committed:
Enhance AI Chat Bot component layout and styling for improved user experience

nitinsonawane123 committed:
Commit Changes for attachement ui issue

nitinsonawane123 committed:
Merge pull request #1802 from Wai-Technologies/users/nitin/bug-aatachement-ui-issue

Resolved Bug : Attachement Dropdown ui issue & AI Pundit Chat added one element.

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/task-1691-Create-new-component-AI-Chat-Bot-as-per-design

nitinsonawane123 committed:
Commit Changes for bug 1711

nitinsonawane123 committed:
Merge branch 'development' into users/nitin/bug-1711-element-offcanvas-size-on-button-is-inconsistent

nitinsonawane123 committed:
commit changes for offcanvas button issue

nitinsonawane123 committed:
commit changes

ashoksinarewai committed:
commit code for text area alignment issue and card size issue

ashoksinarewai committed:
Merge branch 'development' into users/ashok/bug-1718-ai-prompt-box-text-area-alignament-issue-and-card-size-issue

Dipak7028 committed:
Resolved bug 1294: Elements> Grid > Enhancement as per Figma Design

 Resolved bug 1294: Elements> Grid > Enhancement as per Figma Design
CSS Changes don

Dipak7028 committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

AkshayPotphode committed:
Change position of tooltip as per figma design kit

AkshayPotphode committed:
Commit for dark theme button text color change

ashoksinarewai committed:
commit code for grid line css related issue.

AkshayPotphode committed:
Commit for dark theme label color change

nitinsonawane123 committed:
Commit Changes for bug 1792

ashoksinarewai committed:
Merge pull request #1806 from Wai-Technologies/users/ashok/bug/1720-grid-lines-are-not-visible

Resolved Bug 1720 - Grid Line Not Visible In Dark Theme.

ashoksinarewai committed:
Merge pull request #1804 from Wai-Technologies/users/ashok/bug-1718-ai-prompt-box-text-area-alignament-issue-and-card-size-issue

Resolved Bug 1718 - Ai Prompt Box Textarea and Card Issue

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1760-components-application-work-flows-UI-issue-in-small-mobile-view

rameshwarkamandar committed:
Resolved margin issue

nitinsonawane123 committed:
Commit Changes Offcanvas and AI Pundit Component

Dipak7028 committed:
Update rds-comp-cache.tsx

supriyartambewaiin committed:
Merge branch 'development' into users/rameshwar/1701-Create-new-component-Side-Bar-as-per-design

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-1711-element-offcanvas-size-on-button-is-inconsistent

nitinsonawane123 committed:
Merge pull request #1803 from Wai-Technologies/users/nitin/bug-1711-element-offcanvas-size-on-button-is-inconsistent

Resolved Bug 1711 : Design System > Elements > Offcanvas > Size on button is consistent.

supriyartambewaiin committed:
Merge branch 'development' into users/rameshwar/1701-Create-new-component-Side-Bar-as-per-design

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1760-components-application-work-flows-UI-issue-in-small-mobile-view

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-element-radio-button-not-showing-same-as-wireframe

rameshwarkamandar committed:
Merge pull request #1810 from Wai-Technologies/users/rameshwar/1701-Create-new-component-Side-Bar-as-per-design

Bug 1701: Resolved Issue on Design System > Component > Top margin issue

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-element-radio-button-not-showing-same-as-wireframe

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1760-components-application-work-flows-UI-issue-in-small-mobile-view

Dipak7028 committed:
Merge pull request #1798 from Wai-Technologies/users/dipak/bug-1760-components-application-work-flows-UI-issue-in-small-mobile-view

Resolved Bug :1760 Design System Component Application Workflows UI issue in small mobile view

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-element-radio-button-not-showing-same-as-wireframe

ashoksinarewai committed:
commit code  for community file widget css issue.

nitinsonawane123 committed:
Merge pull request #1808 from Wai-Technologies/users/nitin/bug-element-radio-button-not-showing-same-as-wireframe

Resolved bug 1729 : Design System > Elements > Radio Button > Radio Button is different than WF and More space is there in between Radio Button and Label.

AkshayPotphode committed:
Resolve Conversation

ashoksinarewai committed:
Merge branch 'development' into users/ashok/bug-1705-community-file-widget-alignment-issue

supriyartambewaiin committed:
Merge branch 'development' into feature/node_upgrade

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1763-components-cache-long-cache-text-are-hidden-behind-delete-button-in-mobile-views

supriyartambewaiin committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

supriyartambewaiin committed:
Merge branch 'development' into users/akshay/bug-alignment-and-dark-theme-issues

AkshayPotphode committed:
resolve conversation

AkshayPotphode committed:
Merge branch 'users/akshay/bug-alignment-and-dark-theme-issues' of https://github.com/Wai-Technologies/raaghu-react into users/akshay/bug-alignment-and-dark-theme-issues

ashoksinarewai committed:
Merge pull request #1811 from Wai-Technologies/users/ashok/bug-1705-community-file-widget-alignment-issue

Resolved Bug -1705 Community File Widget Alignment Issue

Dipak7028 committed:
Resolved bug 1294: Elements> Grid > Enhancement as per Figma Design

icon name change

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1763-components-cache-long-cache-text-are-hidden-behind-delete-button-in-mobile-views

Dipak7028 committed:
Merge pull request #1793 from Wai-Technologies/users/dipak/bug-1763-components-cache-long-cache-text-are-hidden-behind-delete-button-in-mobile-views

Resolved Bug :1763 Design System Component Cache Long Cache text are hidden behind delete button in mobile views

Dipak7028 committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

supriyartambewaiin committed:
Merge branch 'development' into users/akshay/bug-alignment-and-dark-theme-issues

Dipak7028 committed:
Update rds-comp-grid.tsx

Dipak7028 committed:
Merge branch 'users/rameshwar/1294-Design-System-Component-Design-changes-in-grid' of https://github.com/Wai-Technologies/raaghu-react into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

supriyartambewaiin committed:
Merge branch 'development' into feature/node_upgrade

AkshayPotphode committed:
Merge branch 'development' into users/akshay/bug-alignment-and-dark-theme-issues

AkshayPotphode committed:
Merge branch 'users/akshay/bug-alignment-and-dark-theme-issues' of https://github.com/Wai-Technologies/raaghu-react into users/akshay/bug-alignment-and-dark-theme-issues

AkshayPotphode committed:
Merge pull request #1809 from Wai-Technologies/users/akshay/bug-alignment-and-dark-theme-issues

Resolved Bug 1479, 1481 & 1486 : Alignment And Dark Theme Issues

supriyartambewaiin committed:
Merge branch 'development' into feature/node_upgrade

supriyartambewaiin committed:
Merge pull request #1801 from Wai-Technologies/feature/node_upgrade

Feature/node upgrade

Dipak7028 committed:
CSS change

rameshwarkamandar committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

Dipak7028 committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

rameshwarkamandar committed:
Merge branch 'users/rameshwar/1294-Design-System-Component-Design-changes-in-grid' of https://github.com/Wai-Technologies/raaghu-react into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

rameshwarkamandar committed:
used enum

Sachin-Ghorpade committed:
Committed changes on code connect elements

nitinsonawane123 committed:
Commit Changes for bug 1814

Sachin-Ghorpade committed:
Merge pull request #1813 from Wai-Technologies/users/sachin/bug-1619-design-system-elements-code-connect-changes-for-spinner-and-tabs

Completed Task 1619: Design System > Elements > Code connects changes for-Spinner-Tabs-Text Area-Time Picker-Toggle elements

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-element-chat-header-add-size-control

nitinsonawane123 committed:
Merge pull request #1816 from Wai-Technologies/users/nitin/bug-element-chat-header-add-size-control

Resolved bug 1814 : Design System > Elements > Chat Header > Add size control.

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/task-1691-Create-new-component-AI-Chat-Bot-as-per-design

supriyartambewaiin committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

Dipak7028 committed:
Update rds-comp-faq.tsx

rameshwarkamandar committed:
remove unwanted brackets

rameshwarkamandar committed:
Merge branch 'users/rameshwar/1294-Design-System-Component-Design-changes-in-grid' of https://github.com/Wai-Technologies/raaghu-react into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

rameshwarkamandar committed:
apply Id specific Css

ashoksinarewai committed:
commit code for toggle theme related issue

ashoksinarewai committed:
Merge branch 'development' into users/ashok/bug-toogle-color-and-moon-alignment

nknikhilkasar committed:
Add gap utility class and refactor JSX structure in typing section component

ashoksinarewai committed:
minor changes in css

nitinsonawane123 committed:
Commit Changes for attachement ui

ashoksinarewai committed:
Merge pull request #1819 from Wai-Technologies/users/ashok/bug-toogle-color-and-moon-alignment

Resolved Bug - Togge Theme Color And Moon Alignment

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/task-1646-Create-new-component-Typing-Section-as-per-design--Chat-

Dipak7028 committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

nknikhilkasar committed:
Merge pull request #1818 from Wai-Technologies/users/nikhil/task-1646-Create-new-component-Typing-Section-as-per-design--Chat-

Completed Task 1692: Design System > Component > Create new component Typing Section as per design [Chat]

supriyartambewaiin committed:
Merge branch 'development' into users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

Dipak7028 committed:
Merge branch 'development' into users/dipak/bug-1788-components-Faq--Two-scrollbar-are-present-in-small-mobile-view

Dipak7028 committed:
Merge pull request #1805 from Wai-Technologies/users/rameshwar/1294-Design-System-Component-Design-changes-in-grid

 Resolved bug 1294: Elements> Grid > Enhancement as per Figma Design

supriyartambewaiin committed:
Merge branch 'development' into users/dipak/bug-1788-components-Faq--Two-scrollbar-are-present-in-small-mobile-view

Dipak7028 committed:
Merge pull request #1817 from Wai-Technologies/users/dipak/bug-1788-components-Faq--Two-scrollbar-are-present-in-small-mobile-view

Resolved Bug :1721 Components Faq Two scrollbar are present in small mobile view

nitinsonawane123 committed:
commit changes for ui issue

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-1473-components-review-small-view-ui-issue

nitinsonawane123 committed:
Merge pull request #1820 from Wai-Technologies/users/nitin/bug-1473-components-review-small-view-ui-issue

Resolved bug 1473 : Design System > Component > Review > Enhancement as per design Kit.

nknikhilkasar committed:
Refactor project button click handler in typing section component

supriyartambewaiin committed:
Stable npm version for all environments

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/task-1691-Create-new-component-AI-Chat-Bot-as-per-design

nknikhilkasar committed:
Merge pull request #1771 from Wai-Technologies/users/nikhil/task-1691-Create-new-component-AI-Chat-Bot-as-per-design

Completed Task 1691 : Design System > Component > Create new component AI Chat Bot as per design [Chat]

supriyartambewaiin committed:
Merge branch 'development' into users/supriya/stable-npm-version-for-all-environment

supriyartambewaiin committed:
Merge pull request #1822 from Wai-Technologies/users/supriya/stable-npm-version-for-all-environment

Completed Task: Stable NPM version for all environments.

nknikhilkasar committed:
Refactor CSS for multicolor and premium selected inactive states to improve border styling and background handling

nknikhilkasar committed:
Merge branch 'development' into users/nikhil/task-1704-Create-new-component-Tabs-Free-Premium-as-per-design--Chats

nitinsonawane123 committed:
commit changes for bug 1672

nknikhilkasar committed:
Merge pull request #1824 from Wai-Technologies/users/nikhil/task-1704-Create-new-component-Tabs-Free-Premium-as-per-design--Chats

Completed Task 1704: Design System > Component > Create new component Tabs- Free & Premium as per design [Chat]

ashoksinarewai committed:
commit code for popover alignment and text alignment issue

ashoksinarewai committed:
Merge branch 'development' into users/ashok/bug-1725-popover-alignment-issue

supriyartambewaiin committed:
Merge branch 'development' into users/nitin/bug-1672-element-input-group-button-alignment

AkshayPotphode committed:
Commit changes for profile position change as per figma

nitinsonawane123 committed:
Merge pull request #1825 from Wai-Technologies/users/nitin/bug-1672-element-input-group-button-alignment

Resolved Bug 1672 : Design System > Elements > Fab Menu > After click on fab menu button is not display.

supriyartambewaiin committed:
Resolved Issues on Toast, Scrollbar and pagination and forgot-password

supriyartambewaiin committed:
Merge branch 'development' into users/supriya/bugs/tab-and-scrollbar-issue

supriyartambewaiin committed:
Merge branch 'development' into users/akshay/bug-1698-change-profile-position-as-per-figma

AkshayPotphode committed:
Merge pull request #1827 from Wai-Technologies/users/akshay/bug-1698-change-profile-position-as-per-figma

Resolved Bug 1698 : Profile Position Change as per Figma Design Kit

supriyartambewaiin committed:
pagination alignment

supriyartambewaiin committed:
toast percentage alignment issue

supriyartambewaiin committed:
Merge branch 'development' into users/supriya/bugs/tab-and-scrollbar-issue

supriyartambewaiin committed:
Resolved and add separate CSS for pagination

supriyartambewaiin committed:
Merge branch 'development' into users/ashok/bug-1725-popover-alignment-issue

ashoksinarewai committed:
Merge pull request #1826 from Wai-Technologies/users/ashok/bug-1725-popover-alignment-issue

Resolved Bug - 1725 Popover Text Alignment Issue

nknikhilkasar committed:
Merge branch 'development' into users/supriya/bugs/tab-and-scrollbar-issue

supriyartambewaiin committed:
Merge pull request #1828 from Wai-Technologies/users/supriya/bugs/tab-and-scrollbar-issue

Resolved Multiple Ui Issues

supriyartambewaiin committed:
Merge branch 'staging' into users/supriya/merge-latest-dev-into-stage-for-latest-deployement-of-enhancement

nknikhilkasar reviewed (approved):
Approved
